frontend/src/scenes/userLogic.tsx

```diff
@@ -155,4 +158,12 @@ export const userLogic = kea<userLogicType<UserType, EventProperty, UserUpdateTy
             window.location.href = '/logout'
         },
     }),
+    loaders: {
+        authConfig: [
+            null as AuthConfig | null,
+            {
+                loadAuthConfig: async () => await api.get('api/authentication/'),
+            },
+        ],
+    },
```

Comment:
Similar to the comment on `authentication.py`, this logic is about global instance capabilities, but it complicates `userLogic`.